src/awesome_map/BoundaryInterceptor.js

```diff
@@ -150,6 +154,20 @@ define(function(require) {
          */
         this._easing = options.easing || EasingFunctions.easeOutQuart;
 
+        this._boundarySensitivity = options.boundarySensitivity || DEFAULT_BOUNDARY_SENSITIVITY;
+        /**
+         * Observable for subscribing to scroll events beyond AwesomeMap boundaries.
+         *
+         * @method AwesomeMap#onScrollPastTopBoundary
+         * @param {Function} callback
+         *        Invoked with (sender, {
+         *            boundary: {@link BoundaryTypes}
+         *        })
+         */
+        // @TODO: Does this need to be disposed somewhere?  If so, where?
```

Comment:
I don't know the answer on this one.  It doesn't look like awesome map ever permanently removes the BoundaryInterceptor, and the AwesomeMap and ScrollList don't seem to dispose either.